src/Modal.js

```diff
@@ -166,6 +175,7 @@ const defaultProps = {
   autoFocus: true,
   enforceFocus: true,
   restoreFocus: true,
+  restoreFocusOptions: {},
```

Comment:
can you omit the default or make it null or undefined?

Reply:
Done